create_release.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -ex
+
+git clone -c http.sslVerify=false "https://gitlab-ci-token:$GITLAB_TOKEN@gitlab.cee.redhat.com/mk-ci-cd/managed-kafka-versions.git" managed-kafka-versions
+cd managed-kafka-versions
+BRANCH_NAME="kas-fleet-manager-${VERSION}"
+# only update the config, if different
+CURRENT_COMMIT_SHA=$(yq '.service.scm.commitSha' services/kas-fleet-manager.yaml)
+echo "Checking if the latest commit sha: $LATEST_COMMIT is different than current config commit sha: $CURRENT_COMMIT_SHA"
+if [[ "${CURRENT_COMMIT_SHA}" != "${LATEST_COMMIT}" ]]; then
+  git checkout -b "$BRANCH_NAME"
+  echo "Updating commit sha and image tag for kas-fleet-manager configuration"
+  # update commitSha
+  yq -i ".service.scm.commitSha = \"$LATEST_COMMIT\"" services/kas-fleet-manager.yaml
```

Comment:
Isn't this going to reformat the whole file?

Reply:
its not. see sample MRs in the comments/ description

Reply:
You are right. The formatting occurs only when array are used in the yaml, and this is not the case.